src/main/webapp/i18n/de/aboutUs.json

```
@@ -67,8 +67,12 @@
             },
             "getInTouch": "Kontakt",
             "contributors": "Mitwirkende",
-            "projectManager": "Projekt Manager",
-            "website": "Webseite"
+            "website": "Webseite",
+            "roles": {
+                "projectManager": "Projekt Manager",
```

```suggestion
                "projectManager": "Projektmanagement",
```
Nitpick: No space in compound nouns.
Additionally, ‘Management’ instead of ‘Manager’ to align with ‘Leitung’ and ‘Administration’.

---

I agree :)